apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateQuestion.scss

```diff
@@ -46,11 +51,26 @@
         display: contents !important;
     }
     .create-question__container {
-
+        font-family: Public Sans;
         width: 500px;
-        margin: 28px auto;
+        margin: 14px auto;
```

Comment:
Suggestion: It is highly encouraged to use `rem` over `px` for font sizes. 

In Figma you can update the style display to use `rem` by applying this setting:

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/7aa0413b-dd92-4bea-a139-25af8661db8b)